scrapers/data18.yml

```diff
@@ -0,0 +1,48 @@
+name: data18
+performerByURL:
+  - action: scrapeXPath
+    url: 
+      - data18.com/
+    scraper: performerScraper
+sceneByURL:
+  - action: scrapeXPath
+    url: 
+      - data18.com/content
+    scraper: sceneScraper
+xPathScrapers:
+  performerScraper:
```

Comment:
like what @Belleyy mentioned we should either remove the performer scraper (since it only gets the name) or add more data to it